target/linux/realtek/files-5.10/arch/mips/rtl838x/setup.c

```diff
@@ -28,6 +29,46 @@
 
 extern struct rtl83xx_soc_info soc_info;
 
+void plat_get_frequencies(u32 *cpu, u32 *lxb, u32 *mem)
+{
+	struct device_node *np;
+	struct clk *clk;
+
+	*cpu = 500000000; // fallback
```

Comment:
```suggestion
	*cpu = 500000000; /* fallback */
```
though I'd also add when/why the fallback happens ...

Reply:
I would rather not have a fallback at all. This is code that is meant to run on SoCs that have different CPU clock rates. Either DT correctly specifies the clock rate, or it doesn't and then we can't assume anything (without also checking what platform we're on, which is even worse IMHO).

Reply:
fallback rewritten.